app.py

```diff
@@ -14,7 +17,9 @@ def main_route():
 
     madlibs = madlib_dict['madlibs']
 
-    madlib = madlibs[randint(0, 4)]
```

Comment:
Did you add this for development purposes? Shouldn't it be the full five at this point?